app/concerns/cmr_collections_helper.rb

```diff
@@ -26,4 +26,24 @@ def get_revisions(concept_id, revision_id)
 
     revisions
   end
+
+  def get_associated_services(response)
+    if response.present?
+      service_ids = response.dig(0, 'meta', 'associations', 'services')
+      # This can happen when there are variable associations and probably tool
```

Comment:
I am getting the number of associated variables for cascade delete notifications in a different place. Services is different because you have to get the actual concepts as well, and do it in more places. I'm not sure if anything should change for either PR at the moment, but we might want to take another look when both are in to see if there's a slightly more efficient way of doing both things that are needed.